src/tests/ftest/util/server_utils.py

```diff
@@ -525,78 +579,93 @@ def kill(self):
-        device_type (str): storage type - scm or nvme
+        Raises:
+            ServerFailed: if server failed to prepare storage
 
-    Raises:
-        ServerFailed: if server failed to prepare storage
+        """
+        # Get the daos_server from the install path. Useful for testing
+        # with daos built binaries.
+        dev_param = ""
+        device_args = ""
+        daos_srv_bin = os.path.join(self.daosbinpath, "daos_server")
+        if device_type == "dcpm":
+            dev_param = "-s"
+        elif device_type == "dcpm_nvme":
+            device_args = " --hugepages=4096"
+        elif device_type == "ram_nvme" or device_type == "nvme":
+            dev_param = "-n"
+            device_args = " --hugepages=4096"
+        else:
+            raise ServerFailed("Invalid device type")
+        cmd = "sudo {} storage prepare {} -u \"{}\" {} -f".format(
+            daos_srv_bin, dev_param, user, device_args)
+        result = pcmd(self._hosts, cmd, timeout=120)
+        if len(result) > 1 or 0 not in result:
+            raise ServerFailed("Error preparing NVMe storage")
 
-    """
-    # Get the daos_server from the install path. Useful for testing
-    # with daos built binaries.
-    dev_param = ""
-    device_args = ""
-    daos_srv_bin = get_file_path("bin/daos_server")
-    if device_type == "dcpm":
-        dev_param = "-s"
-    elif device_type == "dcpm_nvme":
-        device_args = " --hugepages=4096"
-    elif device_type == "ram_nvme" or device_type == "nvme":
-        dev_param = "-n"
-        device_args = " --hugepages=4096"
-    else:
-        raise ServerFailed("Invalid device type")
-    cmd = ("{} storage prepare {} -u \"{}\" {} -f"
-           .format(daos_srv_bin[0], dev_param, user, device_args))
-    result = pcmd(hosts, cmd, timeout=120)
-    if len(result) > 1 or 0 not in result:
-        raise ServerFailed("Error preparing NVMe storage")
-
-
-def storage_reset(hosts):
-    """Reset the Storage on servers using the DAOS server's yaml settings file.
-
-    NOTE: Don't enhance this method to reset SCM. SCM will not be in a useful
-    state for running next tests.
+    def storage_reset(self):
+        """Reset the servers' storage.
 
-    Args:
-        hosts (str): a string of comma-separated host names
+        NOTE: Don't enhance this method to reset SCM. SCM will not be in a
+        useful state for running next tests.
 
-    Raises:
-        ServerFailed: if server failed to reset storage
+        Raises:
+            ServerFailed: if server failed to reset storage
 
-    """
-    daos_srv_bin = get_file_path("bin/daos_server")
-    cmd = "sudo {} storage prepare -n --reset -f".format(daos_srv_bin[0])
-    result = pcmd(hosts, cmd)
-    if len(result) > 1 or 0 not in result:
-        raise ServerFailed("Error resetting NVMe storage")
+        """
+        daos_srv_bin = os.path.join(self.daosbinpath, "daos_server")
+        cmd = "sudo {} storage prepare -n --reset -f".format(daos_srv_bin)
```

Comment:
Likewise here, this should not need sudo to run.

Reply:
Thanks - removed `sudo` from the command.